UnitConverter/ViewController.swift

```diff
@@ -51,7 +57,8 @@ class ViewController: UIViewController, UIPickerViewDelegate {
     
     func displayConvertedTemperatureForRow(row: Int) {
         let degreesCelsius = temperatureRange.values[row]
-        temperatureLabel.text = "\(converter.degreesFahrenheit(degreesCelsius))°F"
+        let degreesFahrenheit = converter.degreesFahrenheit(degreesCelsius)
```

Comment:
This ends up being an unnecessary creation of a local constant value, and doesn't add much in terms of clarity or semantics.